### ISSUE TITLE
Changing audio track doesn't change it's name

### ISSUE THREAD

**Issue by IevgenSobko** [staxrip/staxrip#1349]
**Describe the bug**
Changing audio track doesn't change it's name in the main UI.
I often use remuxes with multiple audio languages and noticed that when I change for instance audio track 1 in the main UI to some other track the name remains the same as was for first track.

As a workaroud I'm doing audio inclusion from container options modal window and removing every audio from main UI.

**Comment by Dendraspis** [comment]
You mean by right-clicking and selecting another track/stream?

**Comment by IevgenSobko** [comment]
> You mean by right-clicking and selecting another track/stream?

Yep, exactly.

**Comment by IevgenSobko** [comment]
Using the following example, it is easy to reproduce
https://drive.proton.me/urls/9J4ANT1B1G#XRQM1vMBQ0aR

**Comment by Dendraspis** [comment]
After loading:
![image](https://github.com/staxrip/staxrip/assets/50659978/42e71597-4f62-493d-90f3-e2c57cab7294)

After selecting other files via right-click:
![image](https://github.com/staxrip/staxrip/assets/50659978/dad1bcd2-b00f-4aa7-836d-ebcd743e3484)

**Comment by IevgenSobko** [comment]
Click Edit and check the name in popup. It should be from an old track, not newly selected.
At least this way it works for me.

**Comment by Dendraspis** [comment]
Ahh, now I understand what you mean.  😅 

Hmm, but it was always the case, that it is not overwritten, because also custom set track names would be overwritten.  😕

**Comment by IevgenSobko** [comment]
I believe on selection new track previously it did reassignment to new name parsed from selected track...
But maybe you are correct, and I put here desired instead of what actually was for real.
After further thinking, I'm now recalling cases when sometimes my E-AC-3 tracks had names of TrueHD 🤣(because I changed track in UI)

I'm often selecting different tracks for inclusion and for me it is expected to have name changed once I change track.
At first I would change track and then modify to custom name based on already parsed.
But I found a workaround with Container options audio tab and I actually used to it.
